### PR TITLE
Style tweaks

### PR DESCRIPTION
**Pull request by meshulam**
- Style search box and prev/next buttons to match TempoIQ aesthetic
- Make inline code literals look less dumb
- Improve spacing in method attributes table
- Change navbars from black to slightly darker black
- Color mobile menu bar like the other navbars

### CODE REVIEW

File: source/themes/tempoiq_theme/static/js/theme.js [hunk 1]
```diff
@@ -41,7 +42,27 @@ window.SphinxRtdTheme = (function (jquery) {
             enable : enable
         };
     }());
+
+    var scrollToAnchor = function(href, addToHistory){
+        href = typeof(href) == "string" ? href : jquery(this).attr("href");
+        if(!href) return;
+        var fromTop = 80;
```
Comment (selecsosi) [inline source/themes/tempoiq_theme/static/js/theme.js]: This sets the anchor right at the bottom of the title bar. I set this to 100 locally for some padding off the nav bar. I don't know what you liked better look wise. I'm ready to merge this if this is good

**Comment by selecsosi** [comment]
I also noticed when you switch languages it can be kind of jaring. I'm thinking of adding in something that gets the % the example is down the page, switch languages, remeasure the page and scroll to the the old % offset in the new height.

**Comment by meshulam** [comment]
Good call, I changed it to 100. 

Not sure if a % scroll will work, since you might switch to a language that has a longer snippet above but a shorter one below. The same percentage on the page would put you in a different place. What about just using an anchor link to the example box? Then at least the page would always jump in a predictable way. We can discuss tomorrow.

**Comment by selecsosi** [comment]
duh, we have callbacks when the event that was clicked finishes switching language. Thank you single threaded event handling. I sent a pr to your branch that implements an measured offset, move, reoffset approach that seems to work pretty well. Let me know what you think.

**Pull request merged**